build.ps1

```diff
@@ -1,11 +1,11 @@
 param(
     [ValidateSet("vs2013", "vs2015", "nupkg-only", "gitlink")]
     [Parameter(Position = 0)] 
-    [string] $Target = "vs2013",
+    [string] $Target = "vs2015",
```

Comment:
Are you deliberately upgrading the project to `VC++ 2015`? The documentation would need to be updated if this is the path your planning on taking, See #1983 the previous discussion on the topic.

Reply:
Sorry, no I wasn't I just compiled it in 2015. I will fix the target.